lib/rules/indent.js

```diff
@@ -0,0 +1,828 @@
+/**
+* Forked from: https://github.com/eslint/eslint/blob/v2.13.0/lib/rules/indent.js
+* This modification adds the `VariableDeclaratorOffset` configuration variable
+* that allows specifying how many spaces a multi-line variable declaration should
+* be offset from the normal indentation level. For example:
+*
+* var something,
+*     anotherSomething,
+*     somethingElse;
+*
+* The multi-line variable declaration is indented 4 spaces from the normal indentation level,
+* so VariableDeclaratorOffset should be set to '4'.
```

Comment:
Is this correct? It's confusing that it says four here and the PR description shows an example using `1`. Probably best to put example configuration right here in the comment.